specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/DatabaseEncryptionProtectorRevalidate.json

```diff
@@ -0,0 +1,107 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-08-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/encryptionProtector/{encryptionProtectorName}/revalidate": {
+      "post": {
+        "tags": [
+          "DatabaseEncryptionProtectorRevalidate"
+        ],
+        "description": "Revalidates an existing encryption protector for a particular database.",
+        "operationId": "DatabaseEncryptionProtectors_Revalidate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=37&length=2)

use v3 when possible, same for the rest of the PR

Reply:
@strehan1993 Can you please help to check this one ?

Reply:
@zizw123 we already have an ARM approved swagger PR for this controller: https://github.com/Azure/azure-rest-api-specs/pull/21842/files/e628a578fef355190a2328f0cf446383af98f7ff#diff-63566e10030c180f8c31a899d291c41acea8e92f268b8cb4e0740d97422157d4

for private preview and would not be possible to make changes at this point since the code is already deployed. The approval was taken before deploying the code.

Reply:
@strehan1993 Ok, these comments should be taken with next release as well